.github/workflows/mobile-android_build.yml

```diff
@@ -44,7 +44,7 @@ jobs:
       run: |
         cd mobile
         ./bazelw build \
-          --config=mobile-remote-ci \
+          --config=mobile-remote-release-clang \
```

Comment:
Out of curiosity, why this change?

Reply:
That's because `remote-ci` has [--remote_download_minimal](https://github.com/envoyproxy/envoy/blob/e9366f2b34be537b1b30ed17a4fd75b4715b5990/.bazelrc#L391), which means the Bazel output will not be available.

Reply:
to expand on this, when i last worked on the bazel flags i implemented a pattern - more for Envoy, but also for mobile

Anything that ends `remote-ci` does not download - in the case of envoy the equivalent `--config=remote` + `--config=ci` are the same except that it doesnt enforce a download style